java/api/src/main/java/io/ray/api/Ray.java

```diff
@@ -259,19 +258,11 @@ public static RuntimeContext getRuntimeContext() {
    * for the resource to be updated and rescheduled. This function only works when gcs actor manager
    * is turned on.
    *
-   * @param name Name of the placement group.
-   * @param bundles Pre-allocated resource list.
-   * @param strategy Actor placement strategy.
+   * @param creationOptions Creation options of the placement group.
    * @return A handle to the created placement group.
    */
-  public static PlacementGroup createPlacementGroup(
```

Comment:
API changes should be backward-compatible.

Reply:
Updated.